ggu/main.c

```diff
@@ -47,8 +54,9 @@ filter_name(const struct gatekeeper_if *iface)
 }
 
 static void
-process_single_policy(const struct ggu_policy *policy, const struct ggu_config *ggu_conf)
+process_single_policy(struct ggu_policy *policy, void *arg)
```

Comment:
We need to move the fives calls `rte_be_to_cpu_32()` in the body of this function to `ggu_policy_iterator()` so both, `process_single_policy()` and `print_unsent_policy()` work as expected.